casbin/README.md

```diff
@@ -153,3 +153,63 @@ func main() {
   app.Listen(":8080")
 }
 ```
+## Notes on Config's Enforcer
+
+Use this property instead, if you
+- manage to test out above example with success
```

Comment:
I'm assuming the purpose is to be able to have policies in memory instead of a file?

Reply:
yeah, that's the idea, some instance where the policy is static instead of dynamic, perhaps it's better to hardcode the policy in memory instead of filesystem, at least that's what I'm thinking. because casbin can be used as static or dynamic policy i presumed, if anyone set the filesystem mode too widely then anyone could modify the policy, and maybe wait for the system to crash and restart, then the modified policy will get read.

disclosure, I'm not a security expert, just figure it's safer to just load these in memory instead of files since developer should understand which func call should be protected or not.

Reply:
the example I provided, doesn't use any Auto-Save feature because json-adapter (by casbin folks) is static. for those dynamic adapters with Auto-Save adapter, it's probably for those software system that allows their admin to modify the policy and persist in database(s)